scripts/react-app/opener.js

```diff
@@ -0,0 +1,58 @@
+"use strict";
+let childProcess = require("child_process");
+let os = require("os");
+
+module.exports = function opener(args, options, callback) {
+    let platform = process.platform;
+
+    // Attempt to detect Windows Subystem for Linux (WSL). WSL  itself as Linux (which works in most cases), but in
+    // this specific case we need to treat it as actually being Windows. The "Windows-way" of opening things through
+    // cmd.exe works just fine here, whereas using xdg-open does not, since there is no X Windows in WSL.
+    if (platform === "linux" && os.release().indexOf("Microsoft") !== -1) {
+        platform = "win32";
+    }
+
+    // http://stackoverflow.com/q/1480971/3191, but see below for Windows.
+    let command;
```

Comment:
same here, would be good to extract this til line 30 into a function called `getOpenCommand`